ios/AppLovinMAX.m

```diff
@@ -2321,6 +2321,17 @@ - (void)setAmazonResult:(id /* DTBAdResponse or DTBAdErrorInfo */)result forAdUn
 
         [interstitial setLocalExtraParameterForKey: key value: result];
     }
+    else if ( adFormat == MAAdFormat.rewarded )
+    {
+        MARewardedAd *rewardedAd = [self retrieveRewardedAdForAdUnitIdentifier: adUnitIdentifier];
+        if ( !rewardedAd )
+        {
+            [self log: @"Failed to set Amazon result - unable to find rewarded"];
```

Comment:
```suggestion
            [self log: @"Failed to set Amazon result - unable to find rewarded ad"];
```

Reply:
Also, Android uses `retrieve` while you use `find` here - let's keep the message consistent on both platforms?